Runtime/Cesium3DTileset.cs

```diff
@@ -714,6 +718,29 @@ public bool createPhysicsMeshes
         /// </summary>
         public partial void FocusTileset();
 
+        /// <summary>
+        /// Initiates an asynchronous query for the height of this tileset at a list of positions,
+        /// expressed as longitude and latitude. The most detailed available tiles are used to
+        /// determine each height.
```

Comment:
Changing for consistency with the Unreal documentation for this function

```suggestion
        /// Initiates an asynchronous query for the height of this tileset at a list of 
        /// cartographic positions, where the longitude (X) and latitude (Y) are given in degrees.
        /// The most detailed available tiles are used to determine each height.
```